### PR TITLE
Pass args to analysis server

### PR DESCRIPTION
**Pull request by lukechurch**
Pass:
—dartpad argument

and wire through the sdkPath argument.

TBR @devoncarew @danrubel

### CODE REVIEW

File: lib/src/analysis_server.dart [hunk 1]
```diff
@@ -62,11 +62,21 @@ class AnalysisServerWrapper {
         if (dumpServerMessages) _logger.info('--> $str');
       }
 
-      _init = AnalysisServer.create(
+      final serverArgs = [
+        '--dartpad',
```
Comment (devoncarew) [inline lib/src/analysis_server.dart]: Does the currently specified sdk support this flag? (I know it was added recently).

File: lib/src/analysis_server.dart [hunk 2]
```diff
@@ -62,11 +62,21 @@ class AnalysisServerWrapper {
         if (dumpServerMessages) _logger.info('--> $str');
       }
 
-      _init = AnalysisServer.create(
+      final serverArgs = [
```
Comment (devoncarew) [inline lib/src/analysis_server.dart]: should likely specify the type arg here

File: lib/src/analysis_server.dart (hunk 1 above)
Comment (lukechurch) [reply lib/src/analysis_server.dart]: It's supported as of dev-13, we're now serving against dev-16.

File: lib/src/analysis_server.dart (hunk 2 above)
Comment (lukechurch) [reply lib/src/analysis_server.dart]: thanks, will do.

**Comment by lukechurch** [comment]
Travis is now happy again

**Pull request merged**